src/lib/inputs/InputRadio/InputRadio.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  Radio as MuiRadio,
+  RadioProps,
+  FormControlLabel as MuiFormControlLabel,
+  FormControlLabelProps
+} from '@material-ui/core';
+import classNames from 'classnames';
+import { FormikProps } from 'formik';
+
+export type InputRadioProps<T> = {
+  label: string;
+  labelPlacement?: FormControlLabelProps['labelPlacement'];
+  formikProps?: FormikProps<T>;
+} & RadioProps;
+
+export const InputRadio = ({
+  className,
+  color = 'primary',
```

Comment:
We might need to set the Mui default theme to use our tailwind css variables. Our primary color should be blue instead of the pink that these show up as.